### DISCUSSION TITLE
Hello all Can you help me return the money? I sent Ethereum to buy Manta but I have not received the coins for a month yet

### DISCUSSION THREAD

**Discussion by MOIUJIBKHALIL** [blockscout/blockscout#9435] [Support]
Transaction Hash:
0xcfa120f8493ca438b32e7a3c5cd391e6e5682c587e38562e73b49d6391ce4f98
Status:
Success
Block:
[19028044](https://etherscan.io/block/19028044)
241990 Block Confirmations
Timestamp:
33 days 22 hrs ago (Jan-17-2024 05:19:59 PM +UTC)|Confirmed within 4 secs
Transaction Action:
Transfer0.250315636825044ETH
To
[0x95CeF1...1aC544E5](https://etherscan.io/address/0x95cef13441be50d20ca4558cc0a27b601ac544e5)

From:
[0x144A14231a9c64800E54Bf4EC2f16cbBEaC863D1](https://etherscan.io/address/0x144a14231a9c64800e54bf4ec2f16cbbeac863d1)
To:
[0x95CeF13441Be50d20cA4558CC0a27B601aC544E5](https://etherscan.io/address/0x95cef13441be50d20ca4558cc0a27b601ac544e5)
Value:
0.250315636825044 ETH
$731.91
Transaction Fee:
0.000876775235364 ETH
$2.56
Gas Price:
41.751201684 Gwei (0.000000041751201684 ETH)
![photo_2024-02-20_16-10-58](https://github.com/blockscout/blockscout/assets/159659955/2448634d-c227-40ab-8055-68a92eff4be6)
![photo_2024-02-20_16-11-01](https://github.com/blockscout/blockscout/assets/159659955/a283a9a1-7dd0-4bf4-9e0e-3cebb01b427e)
![photo_2024-02-20_16-11-04](https://github.com/blockscout/blockscout/assets/159659955/6c3073b5-9044-4f68-95f8-b4febca0b079)
![photo_2024-02-20_16-11-11](https://github.com/blockscout/blockscout/assets/159659955/3a85d621-a0d5-480a-a097-fd544b47f83e)
![photo_2024-02-20_16-10-52](https://github.com/blockscout/blockscout/assets/159659955/e7a7dfa3-d8ed-43ff-9fdb-5746c0eedc56)
![photo_2024-02-20_16-11-14](https://github.com/blockscout/blockscout/assets/159659955/acf960a4-67a6-4aa3-a3b3-1d298650fb98)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
